poll-webapp/src/main/resources/locale/portlet/Poll_en.properties

```
@@ -3,4 +3,9 @@
 #####################################################################################
 
 composer.poll.create=Create a Poll
-composer.poll.create.description=Add a Poll to your Activity
\ No newline at end of file
+composer.poll.create.description=Add a Poll to your Activity
+composer.poll.create.question-field=Ask something ...
+composer.poll.create.option-field=Option {0}
+composer.poll.create.add-option=+ Add option
```

```suggestion
composer.poll.create.drawer.option.add=Add option
```
I prefer that you put the "+" directly in vue file and keep the label without the "+"